### DISCUSSION TITLE
Getting GF

### DISCUSSION THREAD

**Discussion by roxas1511** [sleepeybunney/maelstrom#50] [Q&A]
So I just started off the randomizer, I got Ifrit when I was supposed to but I didn't get Siren in dollet. Tried drawing from the boss fight at the tower, and then the Spider fight.
MEch Spider first fight was as it should've been, the second one I fought The Brothers GF, couldn't draw it for them and I didnt get them for defeating them. I was wondering how getting GF's actually works then

**Comment by sleepeybunney** [comment]
It's not 100% consistent right now but generally speaking, the GFs you draw are still on the same _bosses_ (eg. Siren is still on Elvoret) & the GFs you don't draw are still in the same _places_ (eg. Ifrit is still in the Fire Cavern)
